mantidproject/mantid
# Indirect - Bayes - Remove Fit option from plot result menu in Bayes ResNorm interface

The fit option has been remove from the plot result drop-down menu in the indirect Bayes ResNorm interface.

**To test:**
1. Navigate to Interfaces > Indirect > Bayes.
2. Ensure the ResNorm tab is open in the Bayes interface.
3. Click on the 'Plot Result' drop-down menu.
4. Ensure there is no 'Fit' option in the drop-down menu.

Test File:
[irs26173_graphite002_red.zip](https://github.com/mantidproject/mantid/files/1189966/irs26173_graphite002_red.zip)




Fixes #20063. 

**Release Notes** 

Added the change to the release notes.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
